### PR TITLE
Crunchy DB Samples

### PR DESCRIPTION
**Pull request by StevenBarre**
A sample HA crunchy DB with network policies and storage classes and directions on setting up the monitoring stack.

### CODE REVIEW

**Review by mitovskaol — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ You fill find useful docs on how to do things specific to the BCGov's implementa
 
 # Index
```
Comment (gil0109) [inline README.md:7]: Do you want to add some links to documentation here?

General link to docs: https://access.crunchydata.com/documentation/postgres-operator/v5/

I did some additional customization howtos. that I would like to share.. specifically DB tuning changes..

https://access.crunchydata.com/documentation/postgres-operator/5.0.3/tutorial/customize-cluster/
Specifically adding the 

patroni:
  dynamicConfiguration:
    postgresql:
      parameters:
        shared_buffers: 1GB
        work_mem: 2MB

Comment (gil0109) [reply README.md:7]: Another How to, is how to port-forward:

PG_CLUSTER_PRIMARY_POD=$(oc get pod -n <namespace> -o name -l postgres-operator.crunchydata.com/cluster=<clustername>,postgres-operator.crunchydata.com/role=master)


oc -n ae3cec-dev port-forward "${PG_CLUSTER_PRIMARY_POD}" 5432:5432

File: crunchydb/monitoring/README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+# Crunchy DB PGO Monitoring
+
+You can install one Crunchy Monitoring stack in a tools namespace and monitor all your PGO clusters. In these directions replace `LICENSE` with the license plate of your namespaces.
```
Comment (gil0109) [inline crunchydb/monitoring/README.md:3]: Add pre-requisite: Installation of Kustomize is required locally.
https://kubectl.docs.kubernetes.io/installation/kustomize/

File: crunchydb/monitoring/README.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+Finally, create all the items in the manifest.
+
+```bash
+oc -n LICENSE-tools create -f crunchy-monitoring.yaml
+```
+
+## Add monitoring sidecar to your PostgresCluster
+
+Edit your `PostgresCluster` to include the monitoring container as a sidecar
```
Comment (gil0109) [inline crunchydb/monitoring/README.md]: and reinstall cluster using oc install -k ...

File: crunchydb/monitoring/README.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+```yaml
+spec:
+  monitoring:
+    pgmonitor:
+      exporter:
+        image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.0.3-0
+```
+
+Add a `Role` and `RoleBinding` to allow access from your tools namespace where you installed the monitoring.
```
Comment (gil0109) [inline crunchydb/monitoring/README.md]: Add these where your cluster resides (ie. Dev Workspace).

**Review by gil0109 — COMMENTED** [review]
Looks great, just some minor edits if you feel will help ;)

File: README.md (hunk 1 above)
Comment (gil0109) [reply README.md:7]: Another how to, is patch management.  This needs to be managed by each workspace. using these instructions.
https://access.crunchydata.com/documentation/postgres-operator/5.0.3/tutorial/update-cluster/

Comment (StevenBarre) [reply README.md:7]: Added `crunchydb/README.md` with these details

**Review by gil0109 — APPROVED** [review]
Great Job.  Thanks for the edits.

**Pull request merged**